src/scala/main/ly/stealth/mesos/kafka/cli/Cli.scala

```diff
@@ -126,9 +126,19 @@ trait CliUtils
 
       try { response = Source.fromInputStream(connection.getInputStream).getLines().mkString}
       catch {
-        case e: IOException =>
-          if (connection.getResponseCode != 200) throw new IOException(connection.getResponseCode + " - " + connection.getResponseMessage)
+        case e: IOException => {
+          if (connection.getResponseCode != 200) {
+            val br = new BufferedReader(new InputStreamReader(connection.getErrorStream))
```

Comment:
you can use guava's `CharStreams.toString` method here instead of rolling your own.